pkg/upgrade/upgrade.go

```diff
@@ -255,6 +257,10 @@ func CleanupExistingResource(ctx context.Context, cli client.Client, platform cl
 	// Handling for dashboard OdhApplication Jupyterhub CR, see jira #443
 	multiErr = multierror.Append(multiErr, removOdhApplicationsCR(ctx, cli, gvk.OdhApplication, "jupyterhub", dscApplicationsNamespace))
 
+	// cleanup for github.com/opendatahub-io/pull/888
+	deprecatedFeatureTrackers := []string{"kserve-temporary-fixes"}
```

Comment:
As I mentioned a [while back](https://github.com/opendatahub-io/opendatahub-operator/pull/931#issuecomment-2014464759) the name of the `FeatureTracker`, as it's a cluster-wide resource, it's composed from appNs and feature name, so without this change the cleanup actually does not work.

```suggestion
	deprecatedFeatureTrackers := []string{dscApplicationsNamespace + "-kserve-temporary-fixes"}
```

Reply:
@dtrifiro if we can get this piece fixed? it need application namespace as prefix.
we can try have this done by tomorrow in to odh 2.15 release

Reply:
@bartoszmajsak @zdtsw done